src/widgets/settingspages/CommandPage.cpp

```diff
@@ -87,16 +89,16 @@ CommandPage::CommandPage()
     LayoutCreator<CommandPage> layoutCreator(this);
     auto layout = layoutCreator.setLayoutType<QVBoxLayout>();
 
-    auto *view = layout
-                     .emplace<EditableModelView>(
-                         getApp()->getCommands()->createModel(nullptr))
-                     .getElement();
+    view_ = layout
+                .emplace<EditableModelView>(
+                    getApp()->getCommands()->createModel(nullptr))
+                .getElement();
```

Comment:
It's easier to read the diff if this is in the form of
```cpp
auto *view = layout.emplace(...);
this->view_ = view;
```
Then you don't have to change the next lines. (same for the other pages - it makes the whole PR smaller)

Reply:
I dislike the idea of having the same variable have two names in the code. Especially since one of them explicitly through naming convention states that it's a class variable and probably will have impact somewhere else in the code.

Reply:
> Especially since one of them explicitly through naming convention states that it's a class variable

We have a [convention of referring to class members through `this`](https://wiki.chatterino.com/Contributing%20for%20Developers/#this).

Reply:
Good catch, I seem to have been overly aggressive with regex replace